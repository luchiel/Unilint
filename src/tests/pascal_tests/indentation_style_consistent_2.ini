[indentation]
indentation_style = consistent
indentation_policy = by_size
indentation_size = 2
