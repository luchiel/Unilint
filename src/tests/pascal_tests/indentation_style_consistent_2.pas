var
	 i: integer;
  j, k: integer;

begin
  for i := 1 to 10 do begin
    repeat
   writeln(i);
    until k <> j;
    end;
	writeln("done");
end.
