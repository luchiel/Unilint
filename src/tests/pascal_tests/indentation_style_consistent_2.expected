8:1: indent too small
10:1: indent too deep
11:1: spaces expected, tabs found
